Source/objects.cpp

```diff
@@ -3810,11 +3806,11 @@ bool OperateShrineDivine(int pnum, int x, int y)
 		return false;
 
 	if (currlevel < 4) {
-		CreateTypeItem(x, y, false, ITYPE_MISC, IMISC_FULLMANA, false, true);
-		CreateTypeItem(x, y, false, ITYPE_MISC, IMISC_FULLHEAL, false, true);
+		CreateTypeItem({x, y}, false, ITYPE_MISC, IMISC_FULLMANA, false, true);

```

Comment:
Could you run clang-format to make sure that the code follow the project code style.